Makefile

```diff
@@ -176,7 +176,10 @@ run-apk-tests:
 	exit $$_r
 
 run-performance-tests:
-	$(call MSBUILD_BINLOG,run-performance-tests,,Test) $(TEST_TARGETS) /t:RunPerformanceTests
+	_r=0 ; \
+	$(call MSBUILD_BINLOG,run-apk-tests,,Test) $(TEST_TARGETS) /t:RunApkTests /p:RunApkTestsTarget=RunPerformanceApkTests $(APK_TESTS_PROP) || _r=$$? ; \
```

Comment:
Don't set `_r=$$?`.  I know this is tempting, but it's only viable if you do this *once*.  As soon as you do it a *second* time, you introduce the ability to *clear* `$_r`, because e.g. the *second* `msbuild /t:RunPerformanceTests` has an exit value of 0:

    $(call MSBUILD_BINLOG,run-apk-tests,,Test) $(TEST_TARGETS) /t:RunApkTests /p:RunApkTestsTarget=RunPerformanceApkTests $(APK_TESTS_PROP) || _r=$$?
    # sets _r=1
    $(call MSBUILD_BINLOG,run-performance-tests,,Test) $(TEST_TARGETS) /t:RunPerformanceTests || _r = $$?
    # sets _r=0
    exit $$_r;  # look ma, no errors!

As such, you must ensure that *you* control the exit value:

    $(call MSBUILD_BINLOG,run-apk-tests,,Test) $(TEST_TARGETS) /t:RunApkTests /p:RunApkTestsTarget=RunPerformanceApkTests $(APK_TESTS_PROP) || _r=1
    # sets _r=1
    $(call MSBUILD_BINLOG,run-performance-tests,,Test) $(TEST_TARGETS) /t:RunPerformanceTests || _r=1
    # sets _r=1
    exit $$_r;  # look ma, errors!